test/index/types/types.snapshot.cc

```diff
@@ -0,0 +1,103 @@
+  // format-options: showDocs
+  
+  #include "types.h"
+  
+  enum {
+//^^^^ definition [..] $anontype_9a8b4e83cf46cb05_0#
```

Comment:
Does clang have any guarantees about the stability of those hashes from run to run, or version to version?

Reply:
This hash is based on our pinned version of WyHash, not on Clang's code, so it will stay fixed regardless of the run/version. If upgrading WyHash breaks it, we can stay on the existing version, it shouldn't be a problem.